### ISSUE TITLE
Disable Sleep function without changing the 1.5 sec Dialog time?

### ISSUE THREAD

**Issue by pkamb** [binchewer/power_fixer#4]
First, great work here! This improvement is going to make a lot of fat-fingered typists happy.

From [your blog post](http://binchewer.org/blog/index.php?id=1):

> Now, we want to get rid of unwanted unacknowledged sleeping and we want to restore original behavior of showing shutdown options dialog. 
> 
> First task is pretty easy — basically placing `mov rax, 1; retn` in the beginning of handlePowerButtonEvent will do the trick. 
> 
> Second task ... we can reduce timeout from 1.5 seconds to 0.001 seconds.

Is the first task mentioned above actually being done with power_fixer? If so, can you point out where? :)

Background: 

I'm the developer of [PowerKey](https://github.com/pkamb/PowerKey), which enables remapping the Power key to Forward Delete or another key. We've been [searching](https://github.com/pkamb/PowerKey/issues/14) for a 10.9 fix for some time, so many thanks for power_fixer!

Unfortunately the immediate `[Restart | Shutdown | Sleep]` dialog generated by power_fixer'd keypresses, while better than a black Sleep screen, also seems to be unblockable by PowerKey. 

A solution, it seems, would be to stop the Sleep function (via Task One, above?) but leave the 1.5 second dialog lag. PowerKey can then cheerfully respond to sub-1.5 sec presses, and the normal menu pops up after the normal timeout.

**Comment by RSully** [comment]
I agree this would be helpful. See #2.

**Comment by binchewer** [comment]
Hi.

No, it's not implemented yet, and yes, Sleep call could be suppressed, but it is slightly harder than it was with timeout.

I am working on the new version now, and I will keep your project in mind, to probably come up with more elegant solution than to patch everything around. I hope to finish it within a week.

**Comment by pkamb** [comment]
Sounds good! Looking forward to the new version.

**Comment by pkamb** [comment]
Fixed by #8

**Comment by pkamb** [comment]
The OS X 10.9.2 updated re-added support for pre-Mavericks Power key behavior:

```
defaults write com.apple.loginwindow PowerButtonSleepsSystem -bool NO
```

https://github.com/pkamb/PowerKey#additional-steps-for-os-x-109-mavericks

That fixes this issue, but power_fixer is no longer needed.
